governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts

```diff
@@ -1,201 +1,74 @@
 import {
+  AccountType,
+  getPythProgramKeyForCluster,
+  parseBaseData,
   parseMappingData,
   parsePriceData,
   parseProductData,
-  PriceData,
-  ProductData,
+  PermissionData,
+  parsePermissionData,
+  Product,
 } from '@pythnetwork/client'
-import { AccountInfo, Commitment, Connection, PublicKey } from '@solana/web3.js'
-import { Buffer } from 'buffer'
-import { SetStateAction, useContext, useEffect, useRef, useState } from 'react'
+import { Connection, PublicKey } from '@solana/web3.js'
+import assert from 'assert'
+import { useContext, useEffect, useRef, useState } from 'react'
 import { ClusterContext } from '../contexts/ClusterContext'
 import { pythClusterApiUrls } from '../utils/pythClusterApiUrl'
 
 const ONES = '11111111111111111111111111111111'
 
-function chunks<T>(array: T[], size: number): T[][] {
-  return Array.apply(0, new Array(Math.ceil(array.length / size))).map(
-    (_, index) => array.slice(index * size, (index + 1) * size)
-  )
-}
-
-const getMultipleAccountsCore = async (
-  connection: Connection,
-  keys: string[],
-  commitment: string
-) => {
-  //keys are initially base58 encoded
-  const pubkeyTransform = keys.map((x) => new PublicKey(x))
-  const resultArray = await connection.getMultipleAccountsInfo(
-    pubkeyTransform,
-    commitment as Commitment
-  )
-
-  return { keys, array: resultArray }
-}
-
-const getMultipleAccounts = async (
-  connection: Connection,
-  keys: string[],
-  commitment: string
-) => {
-  const result = await Promise.all(
-    chunks(keys, 99).map((chunk) =>
-      getMultipleAccountsCore(connection, chunk, commitment)
-    )
-  )
-
-  const array = result
-    .map(
-      (a) =>
-        a.array
-          .map((acc) => {
-            if (!acc) {
-              return undefined
-            } else {
-              return acc
-            }
-          })
-          .filter((_) => _) as AccountInfo<Buffer>[]
-    )
-    .flat()
-  return { keys, array }
-}
-
-export const ORACLE_PUBLIC_KEYS: { [key: string]: string } = {
+export const MAPPING_ACCOUNT_KEYS: { [key: string]: string } = {
```

Comment:
should this be exposed in pyth-client-js instead and imported here?

Reply:
I think this currently unused so wouldn't mind deleting it, but we could totally move it to pyth-client-js

Reply:
let's delete it then